tests/operators/test_load_file.py

```diff
@@ -1,12 +1,12 @@
 """
-Unittest module to test Agnostic Load File function.
```

Comment:
Please, let's keep this test inside the same path as the operator:
`tests/sql/operators`

I just realized there is a missing s to the directory I created, it would be great if the refactored operators could follow this pattern:
https://github.com/astronomer/astro-sdk/tree/main/tests/sql/operator

Reply:
I'll address this in a different PR, it's already too big.